content/developer/howtos/upgrade/upgrade_util.rst

```diff
@@ -0,0 +1,283 @@
+   :param bool drop_table: if ``True``, drops the source model's table (default: ``True``)
+   :param dict fields_mapping: Dictionary ``{"source_model_field_1": "target_model_field_1", ...}``
+      mapping fields with different names on both models
+   :param list(str) or str ignore_m2m: list of m2m tables ignored to remove from source model.
+
+Modules
+-------
+
+.. `[source] <https://github.com/odoo/upgrade-util/blob/master/src/util/modules.py#L218>`_
+.. method:: remove_module(cr, module)
+
+   Uninstall and remove a module and its references from the database
+
+   :param str module: name of the module to remove
+
+.. `[source] <https://github.com/odoo/upgrade-util/blob/master/src/util/modules.py#L263>`_
+.. method:: rename_module(cr, old, new)
+
+   Rename a module and its references from ``old`` to ``new``
+
+   :param str old: current name of the module to rename
+   :param str new: new name of the module to rename
+
+.. `[source] <https://github.com/odoo/upgrade-util/blob/master/src/util/modules.py#L323>`_
+.. method:: merge_module(cr, old, into, update_dependers=True)
+
+   Move all references of module ``old`` into module ``into``
+
+   :param str old: name of the source module of the merge
+   :param str into: ame of the destination module of the merge
+   :param bool update_dependers: if ``True``, updates the dependencies of modules that depends on
+      ``old`` (default: ``True``)
+
+ORM
+---
+
+.. `[source] <https://github.com/odoo/upgrade-util/blob/master/src/util/orm.py#L43>`_
+.. method:: env(cr)
+
+   Create a new environment from the cursor
+
+   .. warning::
+      This function does NOT empty the cache maintained on the cursor for superuser with an empty
+      environment. A call to invalidate_cache will most probably be necessary every time you
+      directly modify something in database.
+
+   :return: The new environment
+   :rtype: :class:`~odoo.api.Environment`
+
+.. `[source] <https://github.com/odoo/upgrade-util/blob/master/src/util/orm.py#L218>`_
+.. method:: recompute_fields(cr, model, fields[, ids=None][, logger=_logger][, chunk_size=256][, strategy="auto"])
+
+   Recompute field values
+
+   :param str model:  model name of the field(s) to recompute
+   :param list(str) fields: list of field names to recompute
+   :param list(int) ids: list of record IDs to recompute
+   :param logger: Logger used to print the progress of the function
+   :type: logger: :class:`logging.Logger`
+   :param int chunk_size: size of the chunk used to split the records for better processing
+      (default: ``256``)
+   :param str strategy: strategy used to process the recomputation (default: ``auto``):
+
+      - ``flush``: Flush the recomputation when it's finished
+      - ``commit``: Commit the recomputation when it's finished
+      - ``auto``: The function chooses the best alternative for the recomputation based on the
+        number of records to recompute and the fields traceability.
```

Comment:
The bulleted list is broken.

Reply:
It seems like it's not possible to have bulleted lists within params. You can lists the values and their meaning in the body of the docstring instead.